sdk/communication/azure-communication-callingserver/src/test/java/com/azure/communication/callingserver/CallConnectionAsyncTests.java

```diff
@@ -0,0 +1,317 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.callingserver;
+
+import java.util.UUID;
+
+import com.azure.communication.callingserver.models.CallModality;
+import com.azure.communication.callingserver.models.CancelAllMediaOperationsResult;
+import com.azure.communication.callingserver.models.EventSubscriptionType;
+import com.azure.communication.callingserver.models.CreateCallOptions;
+import com.azure.communication.callingserver.models.JoinCallOptions;
+import com.azure.communication.callingserver.models.PlayAudioResult;
+import com.azure.communication.common.CommunicationIdentifier;
+import com.azure.communication.common.CommunicationUserIdentifier;
+import com.azure.communication.common.PhoneNumberIdentifier;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.rest.Response;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+/**
+ * Set the AZURE_TEST_MODE environment variable to either PLAYBACK or RECORD to determine if tests are playback or
+ * live. By default, tests are run in playback mode.
+ */
+public class CallConnectionAsyncTests extends CallingServerTestBase {
+
+    private String from = "8:acs:016a7064-0581-40b9-be73-6dde64d69d72_0000000a-6198-4a66-02c3-593a0d00560d";
```

Comment:
I dont think we should be hardcoding the user ids here. What if someone tries to run these tests with another resource? This would fail with authentication issues

Reply:
Consider creating a new user using the identity package for the tests to make them more robust

Reply:
New user created for testing using identity packages